### PR TITLE
Adding link to "XMPP summits for dummies" in the wiki

### PR DESCRIPTION
**Pull request by winfried**
As discussed at last summit :-)

### CODE REVIEW

**Comment by guusdk** [comment]
Awesome, thanks! Should we add the entire text to the website, instead of to the wiki?

**Comment by winfried** [comment]
@guusdk  Missed your question, sorry. Was considering that too, but because it is a more personal account written from the 'I' perspective, I thought it would be more appropriate to keep the summit for dummies text itself in the Wiki. But I don't object to adding it to the main site.

**Comment by guusdk** [comment]
I'd not mind it being on the main site. I wonder what others think of this.

File: content/pages/community/events.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ __See something missing?__ Events come and go at a speed that is sometimes hard
 
 ## XMPP Summits
 
-The XSF organizes periodic in-person gatherings in order to coordinate high-bandwidth conversations, interop testing, brainstorming and planning, and to conduct other XSF business face-to-face.
+The XSF organizes periodic in-person gatherings in order to coordinate high-bandwidth conversations, interop testing, brainstorming and planning, and to conduct other XSF business face-to-face. Attending a summit for the first time or considering to do so? This may be helpfull: [XMPP summits for dummies](https://wiki.xmpp.org/web/XMPP_summits_for_dummies)
```
Comment (horazont) [inline content/pages/community/events.md]: ``s/helpfull/helpful/``

**Comment by Echolon** [comment]
Hi @winfried,

do you have time to review this again?

**Comment by Echolon** [comment]
Hi @winfried ,

we have been upgrading from Pelican to the Hugo website generator. Could you review this in case it is still relevant?

That would be really great!

**Comment by wurstsalat3000** [comment]
The page is now located at `/content/community/events.md`.

**Comment by wurstsalat3000** [comment]
I took the liberty to add a link to the wiki in #976, because this PR seems to be stuck.

**Pull request closed without merging**